src/Transition.js

```diff
@@ -3,12 +3,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import { timeoutsShape } from './utils/PropTypes';
-
-export const UNMOUNTED = 'unmounted';
-export const EXITED = 'exited';
-export const ENTERING = 'entering';
-export const ENTERED = 'entered';
-export const EXITING = 'exiting';
+import { UNMOUNTED, EXITED, ENTERING, ENTERED, EXITING } from './constants';
```

Comment:
This is also a breaking change.

Reply:
This is only used (imported) in tests and not documented anywhere so I considered these exports part of private API. These however break `babel-add-module-exports` plugin behaviour as it not adding `module.exports = export['default']` at the end of file when there are other named exports present.

Reply:
they are public, if not well documented.

Reply:
I see. They seem to be in conflict with static properties on `Transition` class and that is probably the reason why `babel-add-module-exports` fails to add the statement.

Reply:
the failure there is due to how the transpilation works with a mix of named and default exports. essentially it doesn't want to assign the exports to Transition implicitly. We can "fix" this maybe by manually doing it

Reply:
This is not recommended way as described here: https://github.com/59naga/babel-plugin-add-module-exports/issues/27 and in the referenced issues. As long as the file contains multiple exports this would never work out of the box.

Reply:
right That's why i'm suggesting doing it manually. It's not that it won't work its that it would be destructive for the plugin to do automatically since it might bulldoze over existing properties on the default export

Reply:
Ok, I added a hack to manually insert the cjs default export statement to `Transition`.